inheritance/src/main/java/inheritance/Shop.java

```diff
@@ -0,0 +1,18 @@
+package inheritance;
+
+import java.util.LinkedList;
+
+public class Shop extends Store {
+
+    String description;
+    int price;
```

Comment:
Since the parent `Store` class already has an instance variable called `priceCategory`, you don't need this `int price` here, so just delete it.